src/WhereConstraints/SQLOperator.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Nuwave\Lighthouse\WhereConstraints;
+
+use GraphQL\Error\Error;
+
+class SQLOperator implements Operator
+{
+    public static function missingValueForColumn(string $column): string
+    {
+        return "Did not receive a value to match the WhereConstraints for column {$column}.";
+    }
+
+    public function enumDefinition(): string
+    {
+        return /** @lang GraphQL */ <<<'GRAPHQL'
+enum SQLOperator {
+    EQ @enum(value: "=")
+    NEQ @enum(value: "!=")
+    GT @enum(value: ">")
+    GTE @enum(value: ">=")
+    LT @enum(value: "<")
+    LTE @enum(value: "<=")
+    LIKE @enum(value: "LIKE")
+    NOT_LIKE @enum(value: "NOT_LIKE")
+    IN @enum(value: "In")
+    NOT_IN @enum(value: "NotIn")
+    BETWEEN @enum(value: "Between")
+    NOT_BETWEEN @enum(value: "NotBetween")
+    IS_NULL @enum(value: "Null")
+    IS_NOT_NULL @enum(value: "NotNull")
```

Comment:
@faiverson renamed from `NotNull`, since this matches the actual SQL more closely (`foo IS NOT NULL`). I think clients are more likely to care about that rather then being consistent with the Laravel method that is used.